tmva/tmva/src/MethodDL.cxx

```diff
@@ -1112,11 +1115,14 @@ void MethodDL::TrainDeepNet()
       // initialize the base class pointer with the corresponding derived class object.
       switch (O) {
 
-      // Intentional fall-through
       case EOptimizer::kSGD:
          optimizer = std::unique_ptr<DNN::TSGD<Architecture_t, Layer_t, DeepNet_t>>(
             new DNN::TSGD<Architecture_t, Layer_t, DeepNet_t>(settings.learningRate, deepNet, settings.momentum));
          break;
+      case EOptimizer::kRMSProp:
+         optimizer = std::unique_ptr<DNN::TRMSProp<Architecture_t, Layer_t, DeepNet_t>>(
```

Comment:
can you use `std::make_unique`?

Reply:
I tried. But I got make unique is not declared in this scope. So  I used an alternative.